pkgs/by-name/vo/voicevox-engine/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  replaceVars,
+  voicevox-core,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "voicevox-engine";
+  version = "0.20.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "VOICEVOX";
+    repo = "voicevox_engine";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Gib5R7oleg+XXyu2V65EqrflQ1oiAR7a09a0MFhSITc=";
+  };
+
+  patches = [
+    # the package uses poetry for dependency management, but not package definition
+    # this patch makes the package installable via poetry-core
+    (replaceVars ./make-installable.patch {
```

Comment:
I don't see any vars in the patch, so you should be able to remove `replaceVars` and just use the patch path.

Reply:
@version@ is the var
.
I only supply it for correctness's sake, but it's not strictly required. (It's not used anywhere in the app, so I could just set it to a dummy value)

Reply:
Ah, right. Thank you.